site/docs/components/button/accessibility.mdx

```diff
@@ -11,41 +11,41 @@ data:
 
 ### Best practices
 
-- When a Button has both an icon and text, use`aria-hidden` on the icon so that screen readers do not announce its text.
-- When a Button has an icon only and no text, pass an `aria-label` to Button that describes its purpose.
+- When a button has both an icon and text, use `aria-hidden` on the icon so that screen readers don't announce its text.
+- When a button has an icon only and no text, pass an `aria-label` to the button that describes its purpose.
 
 #### Focusable disabled button
 
-- By default, a disabled button is not focusable. However, in certain scenarios it may be preferable for a disabled button to receive focus. To determine if this is necessary, the following rule of thumb can be followed: "If a disabled element can be enabled (either by the user or by the system) it should be made focusable.” For further information, view the W3C guidance on the [`aria-disabled` attribute](https://www.w3.org/TR/wai-aria/#aria-disabled).
+- By default, a user can't place focus on a disabled button. However in certain scenarios, it may be preferable for a disabled button to receive focus. To determine if this is necessary, you can apply the following rule of thumb: "If the user or the system can enable a disabled element, it should be focusable." For further information, view the W3C guidance on the [`aria-disabled` attribute](https://www.w3.org/TR/wai-aria/#aria-disabled).
 
-#### Screen reader behavior:
+#### Screen reader behavior
 
 On focus, the screen reader should:
 
-- Identify the button
-- Read out the text,
-- Express the state
-  - In certain situations, the screen reader may need to indicate if the button has a popup for lists or menus.
+- Identify the button.
+- Read out the text.
+- Express the state.
+- In certain situations, the screen reader may need to indicate if the button has a popup for lists or menus.
```

Comment:
We should remove this, no other component has it and it's hard to keep up to date and not that useful